modules/administration-guide/partials/proc_configuring-prod-cli-with-environment-variables-to-use-a-rest-backup-server.adoc

```diff
@@ -0,0 +1,18 @@
+[id="configuring-prod-cli-with-environment-variables-to-use-a-rest-backup-server_{context}"]
+= Configuring {prod-cli} with environment variables to use a REST backup server
+
+As an alternative to using command-line options for `{prod-cli} server:backup`, you can configure {prod-cli} for a REST backup server with environment variables that will allow you to run `{prod-cli} server:backup` without having to enter those arguments as options.
+
+.Prerequisites
+
+* Installed `{prod-cli}`.
+* Configured REST backup server. See xref:supported-restic-compatible-backup-servers.adoc[].
+
+.Procedure
+
+* Set the following environment variables:
+** `BACKUP_REPOSITORY_URL` for the backup repository.
+** `BACKUP_REPOSITORY_PASSWORD` for the backup repository password.
+** When authentication is required, `REST_SERVER_USERNAME` for the username for authenticating to the backup REST server.
+** When authentication is required, `REST_SERVER_PASSWORD` for the password for authenticating to the backup REST server.
+** `BACKUP_SERVER_CONFIG_NAME` for the `name` of a `CheBackupServerConfiguration` custom object to point {prod-cli} to. You can find `name` under `metadata` in the custom object. (If you set this environment variable, the preceding four environment variables are not necessary.)
```

Comment:
@missmesss good to know. I've looked it up in IBM SG. I'll keep this as "four" because it just refers to a number of previous points (in-line cross-references, not technical information).